box-share-sdk/src/main/java/com/box/androidsdk/share/fragments/CollaborationsFragment.java

```diff
@@ -61,20 +61,19 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 
         if (savedInstanceState == null) {
             fetchCollaborations();
-        }else if (getArguments() != null) {
+        } else if (savedInstanceState != null) {
+            mCollaborations = (BoxIteratorCollaborations)savedInstanceState.getSerializable(CollaborationUtils.EXTRA_COLLABORATIONS);
+            updateUi();
+        } else if (getArguments() != null){
             Bundle args = getArguments();
```

Comment:
This code will never be executed.

Reply:
Thanks. You're right.  I fixed up the logic now to correctly represent the last known state.